src/components/general/SideSheet/Standard/index.tsx

```diff
@@ -32,6 +32,7 @@ const SideSheet: React.FC<StandardSideSheetProps> = ({
         useComponentDisplayClassNames(styles),
         {
             [styles.isCollapsed]: !isOpen,
+            [styles.isOpen]: isOpen,
```

Comment:
Do we need specific styling for open and collapsed? There are only two states, if it is not collapsed then we know it is open.

Reply:
Romoved one of the states